anda/tools/fast-syntax-highlighting/fast-syntax-highlighting.spec

```diff
@@ -0,0 +1,27 @@
+Name: fast-syntax-highlighting
+Version: 1.67
```

Comment:
1.67 doesn't seem to be a valid version, which is causing the build to fail

Reply:
Yeah the latest is 1.55 I think

you'll also need to use `gh_tag` instead in `update.rhai`

Reply:
Think I corrected the version

Reply:
> gh_tag

Think I did this to